docs/apache-airflow/production-deployment.rst

```diff
@@ -151,7 +151,7 @@ You should be aware, about a few things:
 
 .. code-block:: dockerfile
 
-  FROM apache/airflow:1.10.14
+  FROM apache/airflow:2.0.0
```

Comment:
This tag, `2.0.0`, is not available on Docker Hub. There are only `2.0.0-python3.6`, `2.0.0-python3.7`, and `2.0.0-python3.8`.

There are tags `1.10.12`, `1.10.14`, etc though. So I assume something was missed during the release of 2.0.0?

Reply:
Pushed now :)

Reply:
Good :). I also finalise checking the build scripts on DockerHub and make sure to add it if missing.